controllers/adminController.js

```diff
@@ -66,6 +66,10 @@ const controller = {
       res.status(400).json(error.message);
     }
   },
+  //logout functionality
+  logout: async (req, res) =>{
+    res.clearCookie('token').status(200).json('User Logged-out');
+  },
```

Comment:
response message in json should be small, and no special chars. `user logged out`